backend/Gemfile

```diff
@@ -11,7 +11,7 @@ gem "puma", "~> 5.6"
 gem "redis", "~> 4.0"
 gem "rswag"
 gem "active_storage_validations"
-gem "rspec-rails", "~> 5.0.0"
+gem "rspec-rails", "6.0.0.rc1"
```

Comment:
What is new in rspec 6 that we need rc1 here?

Reply:
Tried to mention this at description, but basically `have_enqueued_mail ` was not working properly. Issue is here: https://github.com/rspec/rspec-rails/issues/2531

Reply:
Ok, thanks.

Reply:
I never used that matcher. Have a bad memory to remember that many of them, :D I think I remember only handful.